image_tools/src/cam2image.cpp

```diff
@@ -113,9 +114,10 @@ int main(int argc, char * argv[])
   // parameter.
   custom_camera_qos_profile.history = history_policy;
 
+  std::cout << "Publishing data on topic " << topic << std::endl;
```

Comment:
I'm not opposed to switching it to printf() (I generally prefer that anyway), but I'm surprised at the rationale.  printf() doesn't guarantee atomicity when writing data out, so it can also be interleaved when multiple threads are in play.  Is the C++ iostream considerably different here?